metaworld/policies/sawyer_pick_place_wall_v2_policy.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+
+from metaworld.policies.action import Action
+from metaworld.policies.policy import Policy, move
+
+
+class SawyerPickPlaceWallV2Policy(Policy):
+
+    @staticmethod
+    @assert_fully_parsed
+    def parse_obs(obs):
+        return {
+            'hand_xyz': obs[:3],
+            'puck_xyz': obs[3:-3],
+            'goal_vec': obs[-3:]
```

Comment:
can we rename this to something else? I feel like calling it goal_vec gives the impression that we're passing the goal coordinate through the observation, when in reality we're passing a vector to the goal from the hand.
@haydenshively wdyt?

Reply:
I honestly think it’s fine as is. Although maybe if the other ones were `hand_pos` and `puck_pos` instead of `_xyz`, it’d be more obvious that when we say `_vec` we really do mean vector and not position.

We should keep a list of things like this and go back through to standardize once the policies are written.

Reply:
I suggest you keep that list on the issue tracker :)